[Backend] As a user, I can upload CSV file of keywords (#57)
Lahphim/go-crawler-challenge: README.md

$ make test
````

#### Background Tasks

> Using [tasks](https://beego.me/docs/module/toolbox.md#tasks) which is providing in the BeeGo's [toolbox](https://beego.me/docs/module/toolbox.md) module.

Reviewer: Syntax issue:

```suggestion
Using [tasks](https://beego.me/docs/module/toolbox.md#tasks) which is provided in the BeeGo's [toolbox](https://beego.me/docs/module/toolbox.md) module.
```

> The mechanism of this module is very similarly to cron jobs 🍀.

Reviewer 2: ```suggestion
The mechanism of this module is very similar to cron jobs 🍀.
```
```suggestion
The mechanism of this module is very similarly to cron jobs 🍀.
```

> The mechanism of this module is very similarly to cron jobs 🍀.

Reviewer 2: ```suggestion
The mechanism of this module is very similar to cron jobs 🍀.
```

So we can create a task and assign the schedule of time to the task, then we can do whatever after the task is triggered by the time we set.

Initializing all the tasks from here `conf/initializers/task.go` and addressing those tasks within this path: `/tasks/*_task.go`

Example:
Setting up the task to run in every minute (https://beego.me/docs/module/toolbox.md#spec-in-detail).
```golang
searchKeywordTask := SearchKeywordTask{Name: "search_keyword_task", Schedule: "0 * * * * *"}
searchKeywordTask.Setup()

...
```

> Add the task then run all of that.

Reviewer: Do you mean?:

```suggestion
Add the task then all of them will be executed with `StartTask()`.
```
